validator/sawtooth_validator/gossip/permission_verifier.py

```diff
@@ -44,7 +44,8 @@ def __init__(self, permissions, current_root_func, identity_cache):
         self._current_root_func = current_root_func
         self._cache = identity_cache
 
-    def is_batch_signer_authorized(self, batch, state_root=None):
+    def is_batch_signer_authorized(self, batch, state_root=None,
+                                   from_state=False):
```

Comment:
Add a doc string for this.  It's unclear what `from_state` means

Reply:
added